state/machine.go

```diff
@@ -2161,21 +2140,21 @@ func (m *Machine) prepareUpgradeSeriesLock(unitNames []string, toSeries string)
 	prepareUnits := make([]unitStatus, len(unitNames))
 	completeUnits := make([]unitStatus, len(unitNames))
 	for i, name := range unitNames {
-		prepareUnits[i] = unitStatus{Id: name, Status: UnitNotStarted}
-		completeUnits[i] = unitStatus{Id: name, Status: UnitNotStarted}
+		prepareUnits[i] = unitStatus{Id: name, Status: params.UnitStarted}
+		completeUnits[i] = unitStatus{Id: name, Status: params.UnitNotStarted}
 	}
 	return &upgradeSeriesLock{
 		Id:             m.Id(),
 		ToSeries:       toSeries,
 		FromSeries:     m.Series(),
-		PrepareStatus:  NotStarted,
```

Comment:
Why default to Started before the unit has noticed that it needs to run the hook? is it just simpler and more straightforward elsewhere?

Reply:
In my mind there are two definitions possible for "Upgrade Series Started". These are:
_An upgrade starts when a user performs `juju upgrade-series prepare`_
Alternatively
_An upgrade starts when the `pre-series-upgrade` hook runs.

There former is by far the easiest the implement - I went occam's razor - hopefully I don't get cut.